src/test/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/EmailContentTest.java

```diff
@@ -30,7 +34,7 @@ void emailBuilderTest() {
         assertEquals(emailContent.getOriginatingAppId(), ORIGINATING_APP_ID);
         assertEquals(emailContent.getMessageId(), MESSAGE_ID);
         assertEquals(emailContent.getMessageType(), MESSAGE_TYPE);
-        assertEquals(emailContent.getData(), DATA);
+        assertEquals(emailContent.getData(), objectMapper.writeValueAsString(DATA));
```

Comment:
this isn't a good test, as you are relying on the object mapper implementation to essentially test the object mapper implementation.
Would suggest defining a static final string as whatever the objectMapper produces and checking that value.

Also I realise I did this wrong, but would you mind reversing the argument order for all of the asserts in this class. as in (expectedValue, actualValue)

Reply:
Replaced with a string for expected value and reversed paramaters in assert equals.